docs/tutorial/index.rst

```diff
@@ -0,0 +1,62 @@
+Javascript Sandbox Tutorial
+===========================
+
+This is a javascript sandbox tutorial for writing Vumi Go Application.
```

Comment:
We might want to remove the "Vumi Go" or mention that it is possible to write standalone Javascript sandbox applications (that connect to Junebug) too.